linters/terraform-docs/plugin.yaml

```diff
@@ -0,0 +1,21 @@
+version: 0.1
+lint:
+  definitions:
+    - name: terraform-docs
+      files: [terraform]
+      main_tool: terraform-docs
+      description: Generate terraform + tofu module documentation
+      commands:
+        - name: terraform-docs
```

Comment:
The command name should more map what the command does than the name of the tool. So in this case given the formatter nature I would set name to "format"